pages/common/ghost.md

```diff
@@ -0,0 +1,36 @@
+# ghost
+
+> A blogging platform and headless CMS.
+> More information: <https://ghost.org>.
+
+- Install ghost in the current directory:
+
+`ghost install`
+
+- Start an instance of Ghost:
+
+`ghost start`
+
+- Restart the Ghost instance:
+
+`ghost restart`
+
+- Check the system for any potential hiccups when on install or update of Ghost:
+
+`ghost doctor`
+
+- View the logs of a Ghost instance:
+
+`ghost log [name]`
```

Comment:
This is confusing. I guess `[name]` should be `{{name}}` here, as that's the syntax we use for variable parameters.

Reply:
Thanks, added this one